src/components/ProfileDropdown.jsx

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { withStyles } from '@material-ui/core/styles';
+import {
+  Avatar,
+  Container,
+  Divider,
+  Grid,
+  Typography,
+} from '@material-ui/core';
+import Menu from '@material-ui/core/Menu';
+import MenuItem from '@material-ui/core/MenuItem';
+import ListItemIcon from '@material-ui/core/ListItemIcon';
+import ListItemText from '@material-ui/core/ListItemText';
+import NewIcon from '@material-ui/icons/NewReleases';
+import BookmarkIcon from '@material-ui/icons/Bookmark';
+import DoneIcon from '@material-ui/icons/Done';
+import ExitToAppIcon from '@material-ui/icons/ExitToApp';
+
+// hacky way to override Menu style
+const StyledMenu = withStyles({
+  paper: {
+    border: '2px solid #d3d4d5',
+    width: '300px',
+    background: '#f2f5fa',
+  },
+})(Menu);
+
+export default function ProfileDropdown(props) {
+  const { signOut, name, email, photo } = props;
+  const [dropMenu, setDropMenu] = React.useState(null);
+
+  const handleClick = (event) => {
+    setDropMenu(event.currentTarget);
+  };
+
+  const handleClose = () => {
+    setDropMenu(null);
+  };
+
+  return (
+    <Container>
+      <Avatar alt="Remy Sharp" src={photo} onClick={handleClick} />
+      <StyledMenu
+        keepMounted
+        anchorEl={dropMenu}
+        open={Boolean(dropMenu)}
+        onClose={handleClose}>
+        <Grid container spacing={2}>
+          <Grid item xs={12} sm={2}>
+            <Avatar alt="Remy Sharp" src={photo} />
+          </Grid>
```

Comment:
Here when I added style and inspected on the web feature for some reason I was not able to horizontally center the grid contain the avatar that's in the dropmenu, suggestion? At the moment keeping it aligned to the left (how it is) creates a nice render illusion with the current size of the avatar that opens the dropdown menu but if centering is a requirement we can change.